package.json

```diff
@@ -11,7 +11,9 @@
     "db-migrate": "yarn prisma migrate dev",
     "db-seed": "yarn ts-node scripts/seed.ts",
     "db-nuke": "docker-compose down --volumes --remove-orphans",
-    "dx": "cross-env BASE_URL=http://localhost:3000 JWT_SECRET=secret DATABASE_URL=postgresql://postgres:@localhost:5450/calendso run-s db-up db-migrate db-seed dev",
+    "db-setup": "run-s db-up db-migrate db-seed",
+    "db-reset": "run-s db-nuke db-setup",
```

Comment:
So we can easily reset between tests

Reply:
More and more reasons to use `prisma db seed` @KATT